lib/v2/gumroad/index.js

```diff
@@ -0,0 +1,35 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const username = ctx.params.username;
+    const products = ctx.params.products;
+    const url = `https://${username}.gumroad.com/l/${products}`;
+
+    const response = await got(url);
+    const $ = cheerio.load(response.data);
+    const title = $('section.product-content.product-content__row > header > h1').text();
+    const userFullName = $('section.product-content.product-content__row > section.details > a').text();
+
+    const item = [
+        {
+            title,
+            link: url,
+            description: art(path.join(__dirname, 'templates/products.art'), {
+                img: response.data.match(/data-preview-url="(.*?)"/)[1],
+                productsName: title,
+                price: $('div.price').text(),
+                desc: $('section.product-content.product-content__row > section:nth-child(3) > div').html(),
+                stack: $('div.product-info').find('ul.stack').html(),
```

Comment:
This looks a bit weired on `/gumroad/afkmaster/Eve10`
```html
<li><h5>Shader</h5><div>UTS &amp; Poiyomi 7.3</div></li><li><h5>Quest Compatible</h5><div>True</div></li><li><h5>Avatar Dynamics</h5><div>Supported</div></li>
```

Reply:
<img width="1057" alt="image" src="https://user-images.githubusercontent.com/30423976/169656673-ff99aa07-951f-48ab-b2a0-b04d8202a8b8.png">

It looks like this, which is ok.